iquip/apps/builder.py

```diff
@@ -330,10 +332,12 @@ def __init__(
         super().__init__(parent=parent)
         self.name = name
         self.state = self.get_state(argInfo)
+        procdesc = self.get_procdesc(argInfo)
         self.stack = QStackedWidget(self)
+        self.rangeWidget = _RangeScan(procdesc, self.state["RangeScan"])
         self.layout = QGridLayout(self)
         self.layout.addWidget(QLabel(name, self), 0, 0)
-        self.layout.addWidget(self.stack, 1, 1)
+        self.layout.addWidget(self.rangeWidget, 0, 1)
```

Comment:
Could you explain your future plan? I think it might be updated to `self.stack` instead of `self.rangeWidget`.

Reply:
Yes I will change it to `self.stack`.